modules/ui/TrainingTab.py

```diff
@@ -474,13 +475,19 @@ def __create_loss_frame(self, master, row, supports_vb_loss: bool):
             components.entry(frame, 2, 1, self.ui_state, "vb_loss_strength")
 
         # Minimum SNR Gamma
-        components.label(frame, 3, 0, "Min SNR Gamma",
-                         tooltip="Minimum SNR gamma. Can help the model learn details more accurately. 0 disables, 20 maximum, ~5 is the usual setting")
-        components.entry(frame, 3, 1, self.ui_state, "min_snr_gamma")
+        components.label(frame, 3, 0, "Loss Weight Function",
+                         tooltip="Choice of loss weight function. Can help the model learn details more accurately.")
+        components.options(frame, 3, 1, [str(x) for x in list(LossWeight)], self.ui_state, "loss_weight_fn")
+        
+        # Debiased Estimation Loss
```

Comment:
Can we make this only show up when something using the parameter gamma is selected?

Reply:
I looked into CustomTkInter docs, issues and the visible API, there doesn't seem to be an obvious way to do it, maybe I'm wrong.

Reply:
Nero just confirmed that we'd have to redraw part of the UI, imma go ahead and close this.